### PR TITLE
Fixed deprecated stuff

### PR DESCRIPTION
**Pull request by GustavoKatel**
This commit should solve the deprecated warnings at the statusbar. I also updated the repo with the new code provided by the package installer.

Good job with this package.

### CODE REVIEW

**Comment by vigo** [comment]
Thank you

**Comment by GustavoKatel** [comment]
No problem

File: lib/atom-html-preview-view.coffee [hunk 1]
```diff
@@ -102,10 +102,7 @@ class AtomHtmlPreviewView extends ScrollView
     </html>
     """
     iframe = document.createElement("iframe")
-    #Fix courtesy of github user kwaak (https://github.com/webBoxio/atom-html-preview/issues/1/#issuecomment-49639162)
-    #This allows use of relative scripts and css files
-    iframe.setAttribute("sandbox", "allow-scripts allow-same-origin")
-    iframe.src = @getPath()
+    iframe.src = "data:text/html;charset=utf-8,#{encodeURI(text)}"
```
Comment (acusti) [inline lib/atom-html-preview-view.coffee]: This is a regression; it removes the fix for #1 and #7 committed in ce3634cbb01efd8bdc4923a33ee3916a8db019fe.

**Comment by GustavoKatel** [comment]
My bad. Thank you, @acusti

**Comment by acusti** [comment]
:+1:

**Pull request merged**